apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data/ISpannerTransaction.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Google.Cloud.Spanner.V1;
+
+namespace Google.Cloud.Spanner
+{
+    internal interface ISpannerTransaction
+    {
+        Task<int> ExecuteMutationsAsync(List<Mutation> mutations, CancellationToken cancellationToken);
+        Task<ReliableStreamReader> ExecuteQueryAsync(string sql, CancellationToken cancellationToken);
```

Comment:
yeah that should be funneled through.  tunnel vision on session pooling, reliable streaming.